integration/modeshape-jbossas-integration-tests/pom.xml

```diff
@@ -259,6 +283,45 @@
                         </configuration>
                     </execution>
                     <execution>
+                        <id>deploy-es-test</id>
+                        <phase>process-test-classes</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.build.directory}/${jboss.server.root.folder}/modules/org/es-test/2.1.0</outputDirectory>
+                            <overwrite>true</overwrite>
+                            <resources>
+                                <resource>
+                                    <directory>src/test/resources/modules/es-test</directory>
+                                    <include>module.xml</include>
+                                </resource>
+                                <resource>
+                                    <directory>${project.build.directory}/libs</directory>
```

Comment:
if the above `copy-dependencies` goal is configured to only include `elasticsearch` related artifacts, this won't need this entire list of specific artifacts to include. Instead, in can include everything from the `elasticsearchLibs` dir.